src/main/kotlin/view/MainView.kt

```diff
@@ -52,34 +54,49 @@ fun DisplayAlgorithmMenu(name: String) {
                 ImageButton(
                     imageResourceId = image,
                     onClick = {
-                    }
+                        if(image == "FindBridge.svg"){
+                            onClick()
+                            }
+                    }, viewModel
                 )
             }
         }
     }
 }
 
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
-fun ImageButton(imageResourceId: String, onClick: () -> Unit) {
+fun ImageButton(imageResourceId: String, onClick: () -> Unit, viewModel: MenuViewModel) {
     Box(
         modifier = Modifier
             .size(440.dp, 60.dp)
             .padding(1.dp)
-            .clickable { onClick() }
             .background(Color(0x00))
     ) {
-        Image(
-            painter = painterResource(imageResourceId),
-            contentDescription = "Button Image",
-            modifier = Modifier.size(445.dp, 59.dp),
-            contentScale = ContentScale.Crop
-        )
+        if (imageResourceId == "FindBridge.svg") {
+            Image(
+                painter = painterResource(imageResourceId),
+                contentDescription = "Button Image",
+                modifier = Modifier.glowRec(viewModel.isFinded).size(445.dp, 59.dp).onClick(onClick = onClick),
+                contentScale = ContentScale.Crop
+            )
+        }
+        else {
+            Image(
+                painter = painterResource(imageResourceId),
+                contentDescription = "Button Image",
+                modifier = Modifier.alpha(0.2f).size(445.dp, 59.dp).onClick(onClick = onClick),
+                contentScale = ContentScale.Crop
+            )
+        }
```

Comment:
Здесь отличается только модификатор, поэтому давай менять только его
```suggestion
        val modifier = when (imageResourceId) {
            "FindBridge.svg" -> Modifier.glowRec(viewModel.isFinded)
            else -> Modifier.alpha(0.2f)
        }

        Image(
            painter = painterResource(imageResourceId),
            contentDescription = "Button Image",
            modifier = modifier.size(445.dp, 59.dp).onClick(onClick = onClick),
            contentScale = ContentScale.Crop
        )
```